docs/common/istio-ambient-update.adoc

```diff
@@ -0,0 +1,784 @@
+
+For general information about update strategies (InPlace and RevisionBased), see link:../update-strategy/update-strategy.adoc#update-strategy[Update Strategy].
+
+For information about installing Istio in ambient mode, see link:./istio-ambient-mode.adoc#introduction-to-istio-ambient-mode[Istio Ambient Mode].
+
+*Note:* Istio's ambient mode is available starting from Istio version 1.24.0.
+
+'''
+
+[[understanding-versioning]]
+== Understanding versioning
+
+The Sail Operator follows Semantic Versioning for all releases. Semantic Versioning uses a three-part version number in the format X.Y.Z to communicate the nature of changes in each release.
+
+**X (Major version)**::
+Indicates significant updates that might include breaking changes, such as architectural shifts, API changes, or schema modifications.
+
+**Y (Minor version)**::
+Introduces new features and enhancements while maintaining backward compatibility.
+
+**Z (Patch version or z-stream release)**::
+Delivers critical bug fixes and security updates, such as Common Vulnerabilities and Exposures (CVEs) resolutions. Patch versions do not include new features.
+
+'''
+
+[[about-istio-update-process]]
+== About Istio update process
+
+After updating the Sail Operator, update the Istio control plane to the latest supported version. The Istio resource configuration determines how the control plane upgrade is performed, including which steps require manual action and which are handled automatically.
+
+In ambient mode, the upgrade process involves three main components:
+
+* **Istio Control Plane** - Manages configuration and certificate distribution
+* **IstioCNI** - Handles traffic redirection configuration
+* **ZTunnel** - Per-node L4 proxy running as a DaemonSet
+
+The Istio resource configuration includes the following fields that are relevant to the upgrade process:
+
+**spec.version**::
+Specifies the version of Istio to install. Use the format vX.Y.Z, where X.Y.Z is the desired Istio release. For example, set the field to v1.24.4 to install Istio 1.24.4. Alternatively, set the value to an alias such as vX.Y-latest to automatically install the latest supported patch version for the specified minor release.
+
+**spec.updateStrategy**::
+Defines the strategy for updating the Istio control plane. The available update strategies are InPlace and RevisionBased.
+
+*Note:* To enable automatic patch upgrades, set the approval strategy of the Operator to Automatic. When the Operator detects a new patch release and the version field uses the vX.Y-latest alias, the control plane is updated based on the configured updateStrategy type.
+
+[[about-istio-control-plane-update-strategies]]
+=== About Istio control plane update strategies
```

Comment:
This is duplicating the content from Update Strategy doc. If you think something is missing, please add it over there.